plugins/GSdx/res/tfx.fx

```diff
@@ -92,6 +92,7 @@ cbuffer cb0
 	float4 VertexScale;
 	float4 VertexOffset;
 	float2 TextureScale;
+	float4 TextureOffset;
```

Comment:
Don't ask me how. But it is possible to shuffle thing around to only do a single division.